lib/ecto/repo/supervisor.ex

```diff
@@ -58,8 +58,9 @@ defmodule Ecto.Repo.Supervisor do
     end
 
     if Code.ensure_compiled(adapter) != {:module, adapter} do
-      raise ArgumentError, "adapter #{inspect adapter} was not compiled, " <>
-                           "ensure it is correct and it is included as a project dependency"
+      raise ArgumentError,
+            "adapter #{inspect(adapter)} was not compiled, " <>
+              "ensure it is correct and it is included as a project dependency"
```

Comment:
Should I roll back or keep formatter changes? They don't look so bad and I guess we want to apply formatter on codebase as we change it.

Reply:
No formatter changes, please. It makes it harder to review and messes up blame. :)

Reply:
@josevalim true, I tell that to people all the time too but still did it for some reason 🤣. It's fixed!